source/developer-guides/debug.rst

```diff
@@ -0,0 +1,197 @@
+.. _debug:
```

Comment:
Rename 'debug' to 'debugging-with-udk'. Same for file name to 'debugging-with-udk.rst'

Reply:
fixed